crates/graphql/src/schema/objects/feed_event.rs

```diff
@@ -117,7 +120,7 @@ pub struct OfferEvent {
     feed_event_id: String,
     twitter_handle: Option<String>,
     wallet_address: String,
-    bid_receipt_address: PublicKey<BidReceipt>,
+    bid_receipt_address: String,
```

Comment:
If bid_receipt account is received, I index the account address in feed_events table otherwise the offer uuid if instruction received so bid_receipt_address is changed to String to cast uuid as String. If Pubkey is the type then Uuid when parsed to string would return an error as its not 32 byte

Reply:
address or id references?